frontend/src/components/pages/UserActions/ActivityDashBoardBarGraph/storyData/ActivityDashBoardBargraphData.json

```diff
@@ -0,0 +1,58 @@
+[
```

Comment:
Try putting all your storybook data in one .json file. Have a look at how sheena has done the UserReviewDashboard component as an example. There is only a single json file that has multiple objects inside it.